src/test/FuzzerImpl.cpp

```diff
@@ -751,10 +755,28 @@ TransactionFuzzer::initialize()
             PublicKey publicKey;
             FuzzUtils::setShortKey(publicKey, i);
 
+            // Test sponsorships, for the moment creating just one, of account 2
+            // by account 1.
+            bool const createSponsorship = (i == SPONSORED_ACCOUNT);
+
             auto createOp = txtest::createAccount(
                 publicKey, FuzzUtils::INITIAL_ACCOUNT_BALANCE);
 
+            if (createSponsorship)
+            {
+                auto sponsorshipOp =
+                    txtest::beginSponsoringFutureReserves(publicKey);
+                sponsorshipOp.sourceAccount.activate() =
+                    toMuxedAccount(sponsorPublicKey);
+            }
             ops.emplace_back(createOp);
+
+            if (createSponsorship)
+            {
+                auto sponsorshipOp = txtest::endSponsoringFutureReserves();
+                sponsorshipOp.sourceAccount.activate() =
```

Comment:
> There are two problems. First, this doesn't work because the sponsored account has to end the sponsorship rather than the sponsoring account.

Aha, thanks.  I've fixed that (only in my private branch so far).

> Second, this implies that your checks that set-up works don't work either.

The checks work; I just forgot to `ops.emplace_back(sponsorshipOp)`!  So those operations, and hence the checks, weren't being executed.